### PR TITLE
Fix broken links and remove pypi references

### LINKED ISSUE #21: Update broken links in documentation
404 Not found
https://github.com/ethyca/fidesdemo
 linked from https://ethyca.github.io/fidesops/tutorial/installation/

Connect error
http://127.0.0.1:5000/
 linked from https://ethyca.github.io/fidesops/tutorial/installation/ 

Connect error
http://localhost:8000/docs
 linked from https://ethyca.github.io/fidesops/tutorial/installation/

404 Not found
https://ethyca.github.io/api

404 Not found
https://github.com/ethyca/fidesdemo/blob/main/flaskr/fidesops.py
 linked from https://ethyca.github.io/fidesops/tutorial/execute_privacy_request/

404 Not found
https://github.com/ethyca/solon/blob/main/src/fidesops/api/v1/scope_registry.py
 linked from https://ethyca.github.io/fidesops/guides/oauth/ 

404 Not found
https://github.com/ethyca/fidesdemo/blob/main/fides_resources/flaskr_postgres_dataset.yml
 linked from https://ethyca.github.io/fidesops/tutorial/execute_privacy_request/

404 Not found
https://ethyca.github.io/fides/language/resources/policy/

Connect error
http://0.0.0.0:8080/docs
 linked from https://ethyca.github.io/fidesops/guides/masking_strategies/  

404 Not found
https://ethyca.github.io/docs
 Linked from: https://ethyca.github.io/fidesops/guides/configuration_reference

404 Not found
https://pypi.org/project/fidesops/
 linked from https://ethyca.github.io/fidesops/deployment/

404 Not found
https://ethyca.github.io/Makefile
 Linked from: https://ethyca.github.io/fidesops/development/overview/

404 Not found
https://github.com/ethyca/solon/releases
 linked from https://ethyca.github.io/fidesops/development/releases/

404 Not found
https://ethyca.github.io/fides/use_cases/use_case1.md
 Linked from: https://ethyca.github.io/fides/language/resources/policy/

404 Not found
https://ethyca.github.io/fides/use_cases/use_case2.md
  Linked from: https://ethyca.github.io/fides/language/resources/policy/ and 10+ more

404 Not found
https://ethyca.github.io/fides/getting_started/production_deployment.md
Linked from: https://ethyca.github.io/fides/getting_started/manual

404 Not found
https://ethyca.github.io/fides/getting_started/tutorial/tutorial.md
Linked from: https://ethyca.github.io/fides/getting_started/manual

404 Not found
https://ethyca.github.io/fides/cli/cli/configuration-file
Linked from: https://ethyca.github.io/fides/cli/cli

404 Not found
https://ethyca.github.io/fides/cli/evaluate/apply
Linked from: https://ethyca.github.io/fides/cli/evaluate/</a><span> and <a href="#">1 more</a></span></div></td><td class="col-details"><a href="#" class="btn btn-default btn-sm pull-right">Details</a></td></tr>

404 Not found
https://ethyca.github.io/fides/cli/init-db/apply\
Linked from: https://ethyca.github.io/fides/cli/init-db/</a><span> and <a href="#">1 more</a></span></div></td><td class="col-details"><a href="#" class="btn btn-default btn-sm pull-right">Details</a></td></tr></tbody>

### PR DESCRIPTION
**Pull request by iamkelllly**
# Purpose
There were quite a few broken links in the documentation, and broken anchors pointing to the API documentation. This addresses these broken links and removes pypi instructions until #62 is complete. 

# Changes
- [ ] remove references to `SolonValidationError`
- [ ] fix broken links
- [ ] fix anchors to swagger `/api` page

# Ticket
Closes #21

### CODE REVIEW

File: docs/fidesops/docs/guides/configuration_reference.md [hunk 1]
```diff
@@ -124,4 +124,4 @@ Please note: Fidesops will filter out any sensitive configuration variables. The
 - `TASK_RETRY_DELAY`
 - `TASK_RETRY_BACKOFF`
 
-For more information please see the [api docs](/docs#/Config).
+For more information please see the [api docs](/api#operations-tag-Config).
```
Comment (pattisdr) [inline docs/fidesops/docs/guides/configuration_reference.md:127]: It looks like masking strategies guides still have links formatted like this too https://github.com/ethyca/fidesops/blob/main/docs/fidesops/docs/guides/masking_strategies.md

File: docs/fidesops/docs/guides/configuration_reference.md [hunk 2]
```diff
@@ -7,7 +7,7 @@ In this section we'll cover:
 - An example `fidesops.toml` configuration file
 - Reporting a running application's configuration
 
-Take me directly to [api docs](/docs#/Config).
+Take me directly to [api docs](/api#operations-tag-Config).
```
Comment (pattisdr) [inline docs/fidesops/docs/guides/configuration_reference.md:10]: Is this link formatting correct? This style of formatting doesn't work for me in elsewhere - I still don't understand where this is supposed to resolve - 

For example, connection config links in guides:
Screen Shot 2021-11-12 at 3 37 40 PM

Goes here:

Screen Shot 2021-11-12 at 3 42 21 PM

File: docs/fidesops/docs/guides/configuration_reference.md (hunk 1 above)
Comment (pattisdr) [reply docs/fidesops/docs/guides/configuration_reference.md:127]: We also have this `/docs#` style formatting for links in several code comments in the tutorial and the quickstart - they're not live links, but perhaps we should remove them -

**Review by pattisdr — COMMENTED** [review]
@iamkelllly nice removing these obsolete solon references - this style of links is not working for me, but maybe i'm missing something.

File: docs/fidesops/docs/guides/configuration_reference.md (hunk 2 above)
Comment (iamkelllly) [reply docs/fidesops/docs/guides/configuration_reference.md:10]: the swagger docs don't resolve when running mkdocs locally, but it should autoresolve from this `/api#operations-tag-Connections`  to this `https://ethyca.github.io/fidesops/api#operations-tag-Connections -> https://ethyca.github.io/fidesops/api/#operations-tag-Connections`
I just tested it again (had to be sure) and it should be working once it's deployed to github pages

File: docs/fidesops/docs/guides/configuration_reference.md (hunk 1 above)
Comment (iamkelllly) [reply docs/fidesops/docs/guides/configuration_reference.md:127]: Ah awesome callout, i'll remove these too!

File: docs/fidesops/docs/guides/database_connectors.md [hunk 3]
```diff
@@ -29,7 +29,7 @@ Other platforms will be added in future releases.
 
 ## Creating a ConnectionConfig object 
 
-The connection between Fidesops and your database is represented by a _ConnectionConfig_ object. To create a ConnectionConfig, you issue a request to the [Create a ConnectionConfig](/api#operations-Connections-get_connections_api_v1_connection_get) operation, passing a payload that contains the properties listed below. 
+The connection between Fidesops and your database is represented by a _ConnectionConfig_ object. To create a ConnectionConfig, you issue a request to the [Create a ConnectionConfig](/fidesops/api#operations-Connections-get_connections_api_v1_connection_get) operation, passing a payload that contains the properties listed below.
```
Comment (pattisdr) [inline docs/fidesops/docs/guides/database_connectors.md:32]: We can specifically link to the create/update endpoint - `/fidesops/api/#operations-Connections-put_connections_api_v1_connection_put`

File: docs/fidesops/docs/guides/database_connectors.md [hunk 4]
```diff
@@ -131,15 +131,15 @@ PUT api/v1/connection/my-mongo-db/secret?verify=false`
...
+You can verify that a ConnectionConfig's secrets are valid at any time by calling the [Test a ConnectionConfig's Secrets](/fidesops/api#operations-Connections-test_connection_config_secrets_api_v1_connection__connection_key__test_get) operation:
 
 
 ```
 GET /api/v1/connection/application-postgresql-db/test
 ```
 
 Once again, the `test_status` and `failure_reason` properties describe the success or failure of the test. If the test failed,
-you should adjust the ConnectionConfig Secrets properties through additional calls to [Set a ConnectionConfig's Secrets](/api#operations-ConnectionConfigs-put_connection_config_secrets_api_v1_connection__connection_key__secret_put)
+you should adjust the ConnectionConfig Secrets properties through additional calls to [Set a ConnectionConfig's Secrets](/fidesops/api#operations-ConnectionConfigs-put_connection_config_secrets_api_v1_connection__connection_key__secret_put)
```
Comment (pattisdr) [inline docs/fidesops/docs/guides/database_connectors.md:142]: internal portion of this link is broken too - 
`/fidesops/api#operations-Connections-put_connection_config_secrets_api_v1_connection__connection_key__secret_put`

File: docs/fidesops/docs/guides/onetrust.md [hunk 5]
```diff
@@ -7,7 +7,7 @@ In this section we'll cover:
 - How to configure OneTrust request intake and storage
 - How to test the OneTrust integration
 
-API docs for OneTrust are part of the [storage](/api#operations-tag-Storage) module (you'll need to `make server` first).
+API docs for OneTrust are part of the [storage](/fidesops/api#operations-tag-Storage) module (you'll need to `make server` first).
```
Comment (pattisdr) [inline docs/fidesops/docs/guides/onetrust.md:10]: We can probably remove the "you'll need to `make server`" now

File: docs/fidesops/docs/guides/storage.md [hunk 6]
```diff
@@ -8,7 +8,7 @@ In this section we'll cover:
 - How to test your storage destinations
 - How to extend this module to create a new, custom storage destination type
 
-Take me directly to [API docs](/api#operations-Storage-upload_data_api_v1_storage__request_id__post/) (you'll need to `make server` first)
+Take me directly to [API docs](/fidesops/api#operations-Storage-upload_data_api_v1_storage__request_id__post/) (you'll need to `make server` first)
```
Comment (pattisdr) [inline docs/fidesops/docs/guides/storage.md:11]: We can remove the make server instructions

Comment (pattisdr) [reply docs/fidesops/docs/guides/storage.md:11]: For some reason, I think you need to remove the trailing slash off of the end of this -

File: docs/fidesops/docs/guides/storage.md [hunk 7]
```diff
@@ -32,7 +32,7 @@ To configure a StorageConfig, you'll first need to choose a storage destination
 
 ## Configuration
 
-Let's get started. To create a new StorageConfig, use the following endpoint ([API docs here](/api#operations-Storage-put_config_api_v1_storage_config_put/)):
+Let's get started. To create a new StorageConfig, use the following endpoint ([API docs here](/fidesops/api#operations-Storage-put_config_api_v1_storage_config_put/)):
```
Comment (pattisdr) [inline docs/fidesops/docs/guides/storage.md:35]: same here - /fidesops/api#operations-Storage-put_config_api_v1_storage_config_put, no trailing slash

File: docs/fidesops/docs/guides/storage.md [hunk 8]
```diff
@@ -144,7 +144,7 @@ To test that your storage destination works correctly, you may hit the upload en
 
 Keep in mind that OneTrust destinations will need to be tested end-to-end, using the OneTrust interface to approve a test privacy request. 
 
-To upload data to a storage destination of choice ([api docs here](/api#operations-Storage-upload_data_api_v1_storage__request_id__post/)):
+To upload data to a storage destination of choice ([api docs here](/fidesops/api#operations-Storage-upload_data_api_v1_storage__request_id__post/)):
```
Comment (pattisdr) [inline docs/fidesops/docs/guides/storage.md:147]: remove trailing slash /fidesops/api#operations-Storage-upload_data_api_v1_storage__request_id__post

**Review by pattisdr — COMMENTED** [review]
nice work figuring out this discrepancy with how relative paths don't work when referencing an anchor on the rendered html page.  now that I could test those links, I just picked up a few more issues, but it's about ready -

File: docs/fidesops/docs/guides/database_connectors.md (hunk 4 above)
Comment (iamkelllly) [reply docs/fidesops/docs/guides/database_connectors.md:142]: these are weird, i diffed with the appropriate path and it seems to be working, which has no diff. So I dirtied and readded, i pray this is the last time!

**Review by pattisdr — APPROVED** [review]
looks good!

**Pull request merged**